test/src/mouse.spec.ts

```diff
@@ -259,4 +259,96 @@ describe('Mouse', function () {
 
     expect(await page.evaluate('result')).toEqual({x: 30, y: 40});
   });
+  it('should throw if buttons are pressed incorrectly', async () => {
```

Comment:
that looks like a potentially breaking change? was it throwing before? at least let's mark it as a non-breaking feature so that it shows up in the changelog.

Reply:
It wasn't throwing before, but it would no-op. This makes this explicit.

Reply:
I've set this to a fix since this is technically the correct behavior. Is that fine?

Reply:
sgtm!